src/components/Modals/Onboarding/OnboardingTransfer.vue

```diff
@@ -329,6 +343,22 @@ export default Vue.extend({
           });
         });
     },
+    getSelectableOptions(exams: FirestoreAPIBExam[], choice: string) {
+      const selectedExams = exams.map(exam => exam.subject);
+      const selectableOptions: string[] = [];
+      // copy subjectsAP but don't include ones that are already selected
+      for (const subject of subjectsAP) {
```

Comment:
Since we only iterate over `subjectsAP` and not `subjectsIB`, the options for the IB exams are the options for AP exams. Left is this PR's staging site and right is master.
![Screen Shot 2021-03-02 at 5 56 22 PM](https://user-images.githubusercontent.com/54298311/109726355-9b82c100-7b80-11eb-88fc-5c07d40019b8.png)

Reply:
added a parameter for this -- thanks for pointing this out.